src/draco/io/file_writer_utils.cc

```diff
@@ -38,11 +38,10 @@ bool DirectoryExists(const std::string &path_arg) {
   struct stat path_stat;
   std::string path = path_arg;
 
-#if defined(_WIN32) && not defined(__MINGW32__)
+#if defined(_WIN32) && !defined(__MINGW32__)
   // Avoid a silly windows issue: stat() will fail on a drive letter missing the
-  // trailing slash.
-  if (path.size() > 0 && path[path.size()] != '\\' &&
```

Comment:
Acessing path[path.size()] is simply wrong, that is not the last character of the string, it's the null terminator.